app/Actions/Photo/Create.php

```diff
@@ -68,30 +68,43 @@ public function add(NativeLocalFile $source_file, ?AbstractAlbum $album, ?int $f
 	{
 		$this->checkQuota($source_file);
 
+		/** @var InitDTO $init_dto */
 		$init_dto = new InitDTO(
 			parameters: $this->strategy_parameters,
 			source_file: $source_file,
 			album: $album,
 			file_last_modified_time: $file_last_modified_time
 		);
 
-		/** @var InitDTO $init_dto */
+		$pipes = [
+			Init\AssertSupportedMedia::class,
+			Init\FetchLastModifiedTime::class,
+		];
+
+		if ($this->strategy_parameters->import_mode->shall_resync_metadata) {
+			$pipes[] = Init\LoadFileMetadata::class;
```

Comment:
isn't this done twice then ? See Line 103.

Reply:
Just saw, checking to see if it can be removed safely